src/components/Home.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import StarWarsShips from "./StarWarsShips"
+import { BrowserRouter as Router, Route, Link} from 'react-router-dom';
+
+export default class Home extends React.Component {  
+  
+  render(){
+
+    return (
+        <div>
+        <Router>
+          <Link to="/StarWarsShips">Star War Ships</Link>
+          <Route path="/StarWarsShips" component={StarWarsShips} />
```

Comment:
It's generally best practice to only have one router, since this is technically a nested router that can cause unintended side effects (in this case, only the `starWarsShips` route will have the `starWarsShip` component.